spec.ts

```diff
@@ -1,5 +1,52 @@
+import { browser, element, By, $$, $, by, Key} from 'protractor'
+
 describe('Protractor', function () {
-    it('Should be alive', function () {
-        expect(true).toBe(true)
+  it('should have name', async function (){
+    await browser.get('/')
+    let a = $$('movie-card').$$('a')
+    console.log(await a.getAttribute('title'))
     })
-})
\ No newline at end of file
+  })   
+
+  it('should have "raiting" pointer', async function (){
+    await browser.get('/')
+    let movieCardRitingPointer = $$('movie-card').first().$('.text-ellipsis [title]');
+    expect(await movieCardRitingPointer.isDisplayed()).toBe(true);
+  })
+  
+  it('should open appropriate "movie details" page, after click on "name" field', async function(){
+    await browser.get('/');
+    let movieCardTitle = $$('movie-card').get(0).$('.text-ellipsis a');
+    let movieCardHref = await movieCardTitle.getAttribute('href');        
+    await movieCardTitle.click(); 
+
+    expect(await browser.getCurrentUrl()).toEqual(movieCardHref); 
```

Comment:
Only URL validation is not enough for angular apps, please add additional validations